isodatetime/data.py

```diff
@@ -717,7 +717,7 @@ class TimePoint(object):
         "day_of_year", "day_of_month", "day_of_week",
         "week_of_year", "hour_of_day", "minute_of_hour",
         "second_of_minute", "truncated", "truncated_property",
-        "dump_format", "time_zone"
+        "truncated_dump_format", "dump_format", "time_zone"
     ]
 
     __slots__ = DATA_ATTRIBUTES + ["truncated_dump_format"]
```

Comment:
If we are adding `truncated_dump_format` to `DATA_ATTRIBUTES` we no longer need to manually add it in `__slots__` (we will now get a duplicate item) so we can just do `__slots__ = DATA_ATTRIBUTES`

Reply:
Good catch! Just did the change locally, all tests passing, updating the pull request.

Thanks!